dockerFiles/marklogic-server-centos:base

```diff
@@ -31,6 +32,12 @@ RUN yum localinstall -y /tmp/marklogic-server.rpm \
     && yum -y install sudo \
     && yum -y clean all
 
+
+
+###############################################################
+# Copy converters package 
+###############################################################
+COPY ${ML_CONVERTERS} /opt/converters.rpm
```

Comment:
@jkerr5 Let me know if you think there is a better location for the RPM to be stored on the image. I put it here without a lot of deep thought.

Reply:
I would just put it in the base/working dir for the image. Opt is typically for installed stuff.

Reply:
5f93314710e4ffbfc2370d8dcb77e8d3ef923f49 Updated LMK if you had a different place in mind, right now we don't set the WORKDIR: https://docs.docker.com/engine/reference/builder/#workdir and it gets set to root, I could see us setting it in /usr/ potentially. minor point.